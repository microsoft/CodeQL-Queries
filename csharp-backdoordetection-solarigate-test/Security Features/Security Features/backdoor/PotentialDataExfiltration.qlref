experimental/Security Features/backdoor/PotentialDataExfiltration.ql