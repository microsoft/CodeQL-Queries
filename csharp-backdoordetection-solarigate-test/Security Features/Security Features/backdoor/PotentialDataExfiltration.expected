| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:240:26:240:31 | access to parameter inData | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:240:26:240:31 | access to parameter inData | access to parameter inData |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:302:33:302:37 | access to local variable text2 | access to local variable text2 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:315:39:315:43 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:320:33:320:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:333:34:333:38 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:339:35:339:39 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:355:33:355:37 | access to local variable text4 | access to local variable text4 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:379:25:379:30 | access to local variable array3 | access to local variable array3 |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:52:23:52:75 | access to property DomainName | access to property DomainName | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:63:21:63:71 | access to property HostName | access to property HostName | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:64:21:64:40 | access to property UserName | access to property UserName | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:81:14:81:57 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:82:29:82:79 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:83:29:83:76 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:84:29:84:77 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:85:29:85:79 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:86:19:86:62 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
| SunburstSimplified.cs:416:44:416:47 | access to parameter body | System or Network information flows from $@ to $@, and is transmitted to the external system. | SunburstSimplified.cs:150:20:150:49 | access to property Value | access to property Value | SunburstSimplified.cs:416:44:416:47 | access to parameter body | access to parameter body |
